system-tests/e2e-transfer-test/runner/build.gradle.kts

```
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+plugins {
+    java
+}
+
+val jupiterVersion: String by project
+val restAssured: String by project
+
+dependencies {
+    testImplementation(testFixtures(project(":common:util")))
+    testImplementation(testFixtures(project(":launchers:junit")))
+
+    testImplementation("io.rest-assured:rest-assured:${restAssured}")
+    testImplementation("org.assertj:assertj-core:3.21.0")
+    testImplementation("org.awaitility:awaitility:4.1.1")
+    testImplementation("org.junit.jupiter:junit-jupiter-api:${jupiterVersion}")
+}
+
+tasks.getByName<Test>("test") {
+    testLogging {
+        showStandardStreams = true
+    }
+}
```

Missing empty line at EOF

```suggestion
}

```